### DISCUSSION TITLE
I keep getting this error htmlspecialchars(): Argument #1 ($string) must be of type string

### DISCUSSION THREAD

**Discussion by oladejihenry** [coderflexx/laravisit#18] [Q&A]
I just tried installing this package in my application and i am getting this error after following the whole installation instruction
```
htmlspecialchars(): Argument #1 ($string) must be of type string, Coderflex\Laravisit\PendingVisit given
```

**Comment by ousid** [comment]
Hey, @oladejihenry 

Can you make a detailed answer, providing your Laravel and PHP version? Log file maybe?
